lang/java/reef-io/src/main/java/org/apache/reef/io/network/naming/NameClient.java

```diff
@@ -86,7 +85,8 @@ private NameClient(
             new NamingResponseHandler(replyLookupQueue, replyRegisterQueue), codec)),
         null, retryCount, retryTimeout);
 
-    this.lookupClient = lookupClient;
+    this.lookupClient = new NameLookupClient(serverAddr, serverPort, timeout, factory,
+        retryCount, retryTimeout, replyLookupQueue, transport);
```

Comment:
`transport` and `this.transport`: inconsistent use. 
Could you following the reef convention? We have discussed about it.